Week-3/bindcallapply.js

```diff
@@ -0,0 +1,61 @@
+console.log('Code for Bind')
+
+let object1 = {
+    name:'Dharmik',
+    age: 19,
+    votability:function(){
+        if (this.age >18){
+           return console.log(this.name + ' is eligible for voting');
+        }
+        return console.log(this.name + ' is not eligible for voting')
+        }
+}
+
+let newEntry = {
+    name:'Patel',
+    age: 17
+}
+object1.votability();
+const obj2 = object1.votability.bind(newEntry);
+
+obj2();
+
+////////////////////////////////
+
+
+console.log('Code for Call')
+let a = 10;

```

Comment:
If you know that you'll not be modifying the variable value then only use const not let.